### ISSUE TITLE
Impossible to launch Somfy Tahoma configuration

### ISSUE THREAD

**Issue by Voileux78** [iMicknl/ha-tahoma#493]
### Did you read the instructions?

- [X] I have read the [README](https://github.com/iMicknl/ha-tahoma/blob/master/README.md), including the [advanced debugging](https://github.com/iMicknl/ha-tahoma/blob/master/README.md#advanced) section.

### The problem

Hello,
I'm a beginner with HA, and moreover with HACS...
I've intalled the package Somfy Tahoma through HACS (select the integration, press install, block SOMFY TAHOMA appears in Red waiting for restarting, then restart the HA server). The "block" SOMFY TAHOMA appears now in the list of HACS integration. But in contrast with the other packages present in the list, it isn't seen as a new repository,  and I don't have the menus "Install" "Information" "Repository" "Dismis" on the SOMFY TAHOMA block. I've only the "repository" menu. So that, I cannot install the integration SOMFY TAHOMA. And nothing happens...
Could you help me please ?

### What version of this integration (ha-tahoma) has the issue?

2.5.1

### What version of Home Assistant Core has the issue?

core-2021.7.3

### Which gateway / hub do you use?

Tahoma box

### Device model

no access to that page

### Device type

no access to that page

**Comment by iMicknl** [comment]
It sounds like you already installed the integration through HACS.

Now it should be as simple as going to your Integrations page, add new and search for Somfy TaHoma.

**Comment by Voileux78** [comment]
Hi, many thanks, it works fine now... I just forgot the last step "add new". I was expecting a direct pop-up in the list !
Many thanks once more !

**Comment by iMicknl** [comment]
Great to hear! Let us know if you have any feedback after usage :-)

**Comment by Voileux78** [comment]
Yes for sure, I will do ! Sounds good because it seems being much more detailed than the information delivered by the TaHoma box itself , in particular for Atlantic.

Alain ROBIC

**Comment by Voileux78** [comment]
As a feed back, I succeeded to manage properly my heaters (much more efficiently than with Tahoma itself). I can manage them individually, change their mode, change the target temperature, aa well as collectively by batch. Just taking care not to manage too much heaters in the same batch because of a Tahoma limitation of max reception of 10 commands simulteanously.
I completed my goal in managing the heaters.
Regarding the Protexial management, I'm currently installing the unofficial addon Protexial Somfy that sounds promising...
Very great job, congratulations to the developpers !
